shared/config.json

```diff
@@ -79,7 +79,20 @@
         "allow_driving_time": true
     },
     "staticAssets": {
-        "useStaticCsvs": true,
-        "analysisResultsRootUrl": "https://s3-us-west-2.amazonaws.com/encompass-public-data/results-csv/"
+        "appIsStatic": true,
+        "rootUrl": "https://d3qtklqrtque1g.cloudfront.net/",
+        "csv": {
+            "useStaticCsvs": true,
+            "path": "results-csv/"
+        },
+        "representativePoints": {
```

Comment:
I don't think this needs to be in the config. `api.ts` could know this, now?

Reply:
Why have it there when the configuration file is more easily changed? Everything like this should be derived from configuration imo

Reply:
Yeah that is true. What is the benefit of removing these paths from here now though?

Reply:
I could add a comment attribute to each asset type here with what the filename structure is?

Reply:
I'd rather not have the path here at the moment. Unless you think it is actually useful

Reply:
@tetraptych @akegan thoughts?

Reply:
Done. Not sure if this is necessarily better though - open to thoughts on this.

Reply:
doesn't seem bad to have it here, then the root url and path after the root url are in the same place